cli/src/install.sh

```diff
@@ -126,13 +125,12 @@ copy_files() {
     success "Copied completions to ${HOME}/.phylum/completions"
 }
 
-pushd "$(dirname "$0")" >/dev/null
+cd "$(dirname "$0")"
 banner
 create_directory
 copy_files
 patch_bashrc
 patch_zshrc
-popd >/dev/null
 
 success "Successfully installed phylum."
 rc_file=$(set -e; get_rc_file)
```

Comment:
We can remove `set -e` in our command substitutions now. That was only a problem for `/bin/bash` (as pointed out by [SC2311](https://github.com/koalaman/shellcheck/wiki/SC2311))